resources/contribute-blog/_blog-template.md

```diff
@@ -0,0 +1,54 @@
+---
+category: 2025
+date: 2025-mm-dd
+seo:
+    title: SEO optimized title under 50chars
+    description: SEO optimized description for the blog post that can be displayed in search results.
+
+labels:
+    - General
+markdown:
+    editPage:
+        hide: true
+---
+# SEO optimized title under 50chars (Recommend keeping it same as the meta title above for consistency)
+
+
+Congrats on taking the first step toward drafting your blog post! Use this template as a starting point to organize your blog post.
+
+<!-- BREAK -->
+
+Refer to the [Contributing Guide](https://xrpl.org/resources/contribute-documentation) for detailed guidelines and examples.
+
+## Section: Handy Reference
+
+To help you get started quickly, here’s a handy guide to the syntax for some of the most frequently used components in a blog post — perfect for streamlining your writing process.
+
+To include an internal link, use a relative path.  For example, here's a link to the [Contributing Guide](../resources/contribute-documentation/index.md).
+
+To include an external link, use the absolute URL instead of a relative path. For example, [Contributing Guide](https://xrpl.org/resources/contribute-documentation).
+
+To include an unordered list, use the following syntax:
+
+- Item 1
+- Item 2
+
+To include an ordered list, use the following syntax:
+
+1. List item 1
+2. List item 2
+
+
+### Sub-section: Including Images In Your Blog Post
+
+To include an image, use the following syntax:
+
+Store your graphics in the `/blog/img` directory. Embed graphics using the syntax:
+
+![image_description](/blog/img/my_image.png)
+
+For example, ![XRPL Developer Portal](/blog/img/docs-iav3/xrpl-docs-home.png) renders as follows.
+
+### Sub-section: Embedding Videos In Your Blog Post
+
+For instructions to embed a video, refer to the [Contribution Guide](https://xrpl.org/resources/contribute-documentation#videos).
```

Comment:
refer to the Contribute Documentation topic (or, if you insist, call it Contributing Guide, as above).